### ISSUE TITLE
Library-wide minimum accessibility standard for all components

### ISSUE THREAD

**Issue by kabaros** [dhis2/ui#1447]
More details about this ticket is available on [Jira](https://dhis2.atlassian.net/browse/LIBS-551). There are tickets for the individual components which should be tackled individually and a separate PR raised for each component.

**Comment by pranjalkuhikar** [comment]
Hi, can I work on improving accessibility for the Tooltip component?

**Comment by kabaros** [comment]
@pranjalkuhikar most of these accessiblity enhancements were already done now , including the Tooltip - are you interested in accessibility work specifically, or any work in the UI (or other libraries and projects) - I can help identify other good first issues

**Comment by khalilurrrahmanridoykhan** [comment]
Hi @kabaros — I would like to work on the remaining **Switch** accessibility task (LIBS-572).

I checked the repository and could not find an open PR currently addressing Switch accessibility. I plan to inspect the component semantics, keyboard/focus behavior, accessible naming, disabled state, and existing test/story coverage, then submit a focused PR for this component only, as requested in the issue.

Could you please confirm that Switch is still unclaimed and ready to be worked on? If the Jira ticket contains acceptance criteria beyond what is visible publicly, I would appreciate those details before implementation.

**Comment by khalilurrrahmanridoykhan** [comment]
Picking up the Switch component (LIBS-572) — will raise a PR shortly.

**Comment by sankorla315** [comment]
Hi @kabaros Can I work on [FlyoutMenu](https://dhis2.atlassian.net/browse/LIBS-563). Could you kindly confirm that it is still unclaimed & ready to be worked on? If any Jira ticket info about acceptance criteria in addition to what is visible publicly, I would appreciate those details before implementation.

**Comment by sankorla315** [comment]
Picking up FlyoutMenu component(LIBS-563), incoming PR shortly!

**Comment by koiralajay-ui** [comment]
Hi, I’d like to work on LIBS-850 (StatusIcon accessibility) if it’s still available. I checked for an assignee or overlapping PR and didn’t find any current work.

Before starting, could you confirm the intended API? My reading of the ticket is:

- hide the icon from assistive technology when adjacent validation text already communicates the state
- provide an accessible label when StatusIcon is used standalone

Should valid and loading follow the same behavior as error and warning, and do you have a preferred API for this (aria-hidden / aria-label or an existing project convention)?

Happy to take this on once confirmed.
